test/basic-sell.ts

```diff
@@ -294,7 +290,7 @@ describe('BasicSellCommand', () => {
             await expect(executeTrigger(triggerId, new BigNumber(targetRatio), triggerData)).not.to.be.reverted
         })
 
-        it('does not recreate the trigger if `continuous` is set to false', async () => {
+        it('clears trigger if `continuous` is set to false', async () => {
             const executionRatio = correctExecutionRatio
```

Comment:
I'd copy paste the rest of the test from `basic-buy.ts`

Reply:
I edited only tests that where affected. let's keep diff minimal